charts/rancher-cis-benchmark/2.0.5-rc2/Chart.yaml

```diff
@@ -0,0 +1,22 @@
+annotations:
+  catalog.cattle.io/auto-install: rancher-cis-benchmark-crd=match
+  catalog.cattle.io/certified: rancher
+  catalog.cattle.io/display-name: CIS Benchmark
+  catalog.cattle.io/kube-version: '>=1.16.0-0'
```

Comment:
Please update this upstream to the format '>= 1.16.0-0 < 1.XX.0-0' where XX is the max Kubernetes version the chart support +1. More on the reasoning here: https://github.com/rancher/charts/pull/1924